apps/hash-api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -455,19 +454,22 @@ export const removeEntityEditorResolver: ResolverFn<
 const parseGqlAuthorizationViewerInput = ({
```

Comment:
This function is the only location where `objectId` and `subjectId` are helpful (so far) and this function will be removed as the types will come directly from GraphQL. (just flagging, not feeling strongly but also not 100% convinced)